src/components/Newbeer/NewBeer.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react'
+import NavBar from '../Navbar/NavBar';
+import axios from 'axios';
+
+export default class NewBeer extends Component {
+
+    state = {
+      name: '',
+      tagline: '',
+      description: '',
+      first_brewed: '',
+      brewers_tips: '',
+      attenuation_level: '',
+      contributed_by: ' '
+    }
+
+    changeHandler = (e) => {
+        let {name, value} = e.target
+        this.setState({
+            [name]: value
+        })
+    }
+
+    submitHandler= (e) => {
+    e.preventDefault()
+    const {name, tagline, description, firstBrewed,  brewersTips, attenuationLevel, contributedBy} = e.target
```

Comment:
You already have the correct values of the inputs in the state, these are the values that you should send